tasks/main.yml

```diff
@@ -1,2 +1,65 @@
----
-# tasks file for .
\ No newline at end of file
+- name: Set database to mysql
```

Comment:
There is a sympa variable that allows to select the database, please check this variable and only run `mysql`-related tasks if it is selected. Or adapt the documentation to reflect that only `mysql` is supported.